app/views/services/index.njk

```diff
@@ -54,31 +52,21 @@
       </div>
     {% endif %}
 
-    <h1 class="govuk-heading-l govuk-!-margin-bottom-6 service-count">
-      {% if services_singular %}
-        You have {{services.length}} service
-      {% else %}
-        You have {{services.length}} services
-      {% endif %}
-    </h1>
```

Comment:
Do we want to show number of services somewhere on this page ?

Reply:
in the design we don't